### ISSUE TITLE
error/bug "no uncommited changes" when integrating opauth plugin into cakephp

### ISSUE THREAD

**Issue by AndanteC** [uzyn/cakephp-opauth#18]
Ok, here's the problem and the solution:

If you are integrating uzyn/cakephp-opauth plugin into dir app/Plugin/Opauth, be sure to delete the .git file located at \app\Plugin\Opauth\Vendor\Opauth.git or you will not able to commit any changes.

**Comment by uzyn** [comment]
Did you run `git commit` from within the Plugin or Vendor dir?

You should probably use `git submodule` or `composer`.

**Comment by AndanteC** [comment]
Hi uzyn:

I'm using Github on Windows since I'm new to git. Not sure if I'm runing "git commit" from or within the vendor dir.
The opauth plugin I used for testing is from your http://opauth.org/file.php?name=Opauth-CakePHP-sample-app.zip cakephp example. I copied and pasted the plugin into the app/Plugin/Opauth in the beginning. It should be the process that brought the entire plugin along with the .git file ("gitdir: ../../../../.git/modules/Plugin/Opauth/modules/Vendor/Opauth"). Btw, your opauth is awesome!
